src/main/java/steps/VehicleTechnicalRecordsSteps.java

```diff
@@ -356,7 +356,7 @@ public void waitForVehicleTechRecordsToBeUpdated(String vin, int seconds) {
 
                 if (status == 200 && recordsNumber > 1) {
                     try {
-                        Thread.sleep(2500);
+                        Thread.sleep(5000);
```

Comment:
isn't it better to increase the number of attempts instead of increasing the sleep ?